### PR TITLE
Update Legacy Documentation

### PR DESCRIPTION
**Pull request by ademidun**
Preview Here: https://app.gitbook.com/s/-MhyEZtd5TVytPJtyS7v-2910905616/~/revisions/JHckyrHlugPul4GCJfW3/

(note that the preview link is not completely accurate because it's showing some files that have been deleted)

For example, Other Tools has been deleted but it's still showing in the Gitbook Preview page.

### CODE REVIEW

File: build/workshops.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 Contract proxy kit and Safe apps as well as the Safe apps SDK have been presented at several events.
```
Comment (germartinez) [inline build/workshops.md:3]: This paragraph should be updated as it is a bit empty in content. Mentioning the CPK should be removed from here as it is not relevant nowadays

File: build/workshops.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
 
-* [ETHOnline 2020 - Introduction to building with Safe Apps SDK & Contract Proxy Kit](https://www.youtube.com/watch?v=YGw8WfBw5OI)
-* [TruffleCon 2020 - Leverage Smart Contract Accounts for Your Dapp](https://www.youtube.com/watch?v=J_3BpQGTLeI)
-* [ETHDenver 2021 - Technical Introduction to the Safe Apps SDK](https://www.youtube.com/watch?v=QG94GpCE0p0)
-* [Hello Decentralization 2021 - Technical Introduction to the Safe Apps SDK](https://www.crowdcast.io/e/hello-decentralization/49)
-
+- [ETHTokyo 2023 - How to Make a Group Wallet to Buy Things with Frens using Safe{Core} Account Abstraction SDK](https://www.youtube.com/watch?v=Vgfgr-oNLMk&t=8360s) ([slides](https://docs.google.com/presentation/d/1menJOy_Hk3dS89VmpDU_ZxJcM90Set3ozNRZgBYJWy8/edit))
```
Comment (germartinez) [inline build/workshops.md:5]: 2022 was a year with where Safe did a lof of talks/workshops that are not included here

Comment (ademidun) [reply build/workshops.md:5]: Where can I find these talks/workshops?

File: build/workshops.md (hunk 1 above)
Comment (ademidun) [reply build/workshops.md]: Removed: https://github.com/safe-global/safe-docs/pull/196/commits/18421d583424821dbfa6aab3c673087a96d58dc1

File: build/workshops.md (hunk 2 above)
Comment (germartinez) [reply build/workshops.md:3]: I was thinking about these two, but I was checking them again and some things have changed over time:

- How Safe modules work: https://www.youtube.com/watch?v=QdOfuxxXVBA
  Only thing here is that the slides have the old logo

- Overview on the Core SDK, modules and guards: https://www.youtube.com/watch?v=ppwE9GXC5MA
  However, the packages inside the Core SDK have changed, so not ideal. :(

**Review by germartinez — APPROVED** [review]

File: build/workshops.md (hunk 2 above)
Comment (ademidun) [reply build/workshops.md]: Added: https://github.com/safe-global/safe-docs/pull/196/commits/82901ed8ad0bd6b1cbd7e6fd6d491ac1afb54766

**Pull request merged**